### DISCUSSION TITLE
How Do I Filter Only the Different Folders?

### DISCUSSION THREAD

**Discussion by tatihulot** [WinMerge/winmerge#1813] [General]
Hi everyone.  WinMerge does exactly what I want.  I want it to compare two folders, and display the difference.
For example, let's say I have a folder named Test C.  It contains the following subfolders:
A
B
C
X 
Y
Z

Let's say my second folder is named Test D.  It contains the following subfolders:
A
B
C
D
E
W
X
Y
Z

WinMerge will compare the two folders, and highlights the D, E, and W folders.  That is EXACTLY what I want.
![testcd](https://user-images.githubusercontent.com/131415372/233513681-4cb298de-0180-4590-980c-e3211238a5c9.jpg)

But I haven't figured out how to filter WinMerge so that it shows ONLY the D, E, and W folders.  Is it possible to display only the highlighted folders in the compare?  
If so, what is the procedure, please?  Thank you.

**Comment by sdottaka** [comment]
Upon looking at the attached image, it appears that the "Include subfolders" option is disabled. If this option is enabled, files within subfolders such as A or B will be compared and their identity will be indicated by an icon, whether they are identical or not.

After enabling the "Include subfolders" option as follows, re-compare and uncheck the "View→Show Identical Items" menu item, and folders identified as identical will be hidden. If you only want to display folders that exist in one location, uncheck the "View→Show Different Items" menu item.

![image](https://user-images.githubusercontent.com/98126/233642720-50e06dcc-ee37-4025-b054-becd8ee59816.png)
![image](https://user-images.githubusercontent.com/98126/233642552-ab89f5e6-a372-43cd-9dce-8ebc2c12b2e5.png)

**Comment by tatihulot** [comment]
Thanks for replying.
I'm having a lot of difficulty understanding this program.  

I copied a folder to my external hard drive.  Obviously, both folders should be exactly the same immediately after I did that.

The folder on my D drive is D:\MOVIE COLLECTION.
The folder on my external hard drive is E:\MOVIE COLLECTION.

After I did this, I added a few folders to D:\MOVIE COLLECTION.   I did not copy them to the external hard drive.

I then ran WinMerge.  I enabled the Include Subfolders option.  

I do not see the new subfolders in D:\MOVIE COLLECTION.    They're not there.

I do not understand why not.  What am I missing or doing wrong?

**Reply by sdottaka** [reply]
Please try the following:

- Check all "Show * Items" menu items under the View menu.
- Check "Include unique subfolders contents" in the "Compare/Folder" category in the Options window.
- Check "Tree Mode" menu item under View menu.
- Perform the comparison again.

![image](https://user-images.githubusercontent.com/98126/233815000-33ec1407-6fe7-43c6-a383-4946dd5954de.png)
![image](https://user-images.githubusercontent.com/98126/233814967-6da5aa1f-4122-435d-bf35-03ec4b2881eb.png)

**Comment by tatihulot** [comment]
Thanks for replying again.  That worked.
But...please let me say this:  I'm not really interested in the contents of the subfolders.  I'm only interested in the names of the subfolders themselves.  I hope that makes sense.  
So, can I configure WinMerge so that it displays only the subfolders within the main folder, so that WinMerge won't compare the contents within the subfolders?  I hope I worded that question correctly.  Thank you.

**Reply by sdottaka** [reply]
If the 'Include subfolders' option is disabled, subfolders will not be compared, in which case, unfortunately, the filter menu item in the View menu will not work for subfolders.
